stable/atlantis/templates/configmap-config.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.config -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "atlantis.fullname" . }}
```

Comment:
This shouldn't be called the name of the chart. It needs to be `-yaml-config` or something like that. Then we can have multiple config maps (we do already) and it's not confusing which each one is for.

Reply:
Oh... sorry, I don't thought about that. It will be changing really easy.